### DISCUSSION TITLE
Billing

### DISCUSSION THREAD

**Discussion by BALYzs** [community/community#64079] [New to GitHub]
Hi! Im one of the app supports from a company and I manage the git account, but in the billing, I would need a bill that contains the tax number of the company. Is there a way to request a bill with specific details on it for the company? 
Thanks in advance

**Comment by queenofcorgis** [comment]
Hi @BALYzs ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
